### ISSUE TITLE
TLS Protocol Version errors on Travis

### ISSUE THREAD

**Issue by elliotweiser** [geerlingguy/mac-dev-playbook#60]
I noticed [Travis CI errors](https://travis-ci.org/geerlingguy/mac-dev-playbook) when installing dependency roles from Ansible Galaxy. The full error message:

```
[ERROR]: failed to download the file: Failed to validate the SSL certificate
for github.com:443. Make sure your managed systems have a valid CA certificate
installed. You can use validate_certs=False if you do not need to confirm the
servers identity but this is unsafe and not recommended. Paths checked for this
platform: /etc/ssl/certs, /etc/ansible, /usr/local/etc/openssl. The exception
msg was: [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version
(_ssl.c:590).
```

I believe this is because GitHub/Galaxy dropped support for TLS 1.0 and MacOS's system Python relies on an old `openssl` library.

I've proposed a workaround to this error here:  https://github.com/geerlingguy/ansible-role-homebrew/pull/91

**Comment by geerlingguy** [comment]
Yeah, haven't had time to take a look until just now (keep getting the 'failed' notifications every day after the Travis CI cron runs though!). I'll take a look.

**Comment by geerlingguy** [comment]
Merged that PR in the upstream role. It works for me... and would be nice to test on at least 10.13 and 10.12 (n and n-1) for this playbook too.

**Comment by elliotweiser** [comment]
Thanks for doing that! I can look into rolling the multi-os testing into this, too (might need a couple days).

**Comment by geerlingguy** [comment]
This was fixed in #61 / by updating the tests to work with High Sierra by default. Let's open a separate ticket if/when we want to test on latest and latest-1 releases of macOS (for now, I'm happy to just do latest, since that's all I'm using now on all my Macs :) ).
